lms/djangoapps/philu_api/views.py

```diff
@@ -119,11 +119,11 @@ def mark_user_chat_read(request):
     return JsonResponse(response.json())
 
 def get_user_data(request):
-    """ Get the user profile data for current user specific user """
+    """ Get the user profile data from nodebb for current user specific user """
```

Comment:
I think this should be ok "Get the user profile data from NodeBB for current user"